caikit/runtime/utils/servicer_util.py

```diff
@@ -292,16 +292,15 @@ def validate_data_model(
                     )
                     continue
 
-                field_message_type = input_proto_msg.fields_by_name[
-                    field.name
-                ].message_type
+                field_descriptor = input_proto_msg.fields_by_name[field.name]
```

Comment:
I think this same better check could be done without changing `field_message_type` and simply adding `field_message_type` to the first clause of the `if` (slightly smaller delta)

Reply:
Good call! I'm not sure I completely understand but I updated to reduce the delta